File: OOP_Variable/2.rb
class C 
    def initialize(v)
        @value = v
    end
    def show()
        p(@value)
    end
    def getValue()
        return @value
    end
    def setValue(v)
        @value = v
    end
end

c1 = C.new(10) #루비에서는 외부에서 인스턴스 변수 접근 불가능
p(c1.getValue())
c1.setValue(20)
c1.show()

File: OOP_Variable/2.py
class C(object) :
    def __init__(self, v) :
        self.value = v
    def show(self) :
        print(self.value)
    def setValue(self, v) :
        self.value = v
    def getValue(self) :
        return self.value

c1 = C(10)
print(c1.getValue()) #함수 밖에서 인스턴스 변수 접근이 가능
c1.setValue(20)
print(c1.getValue())
c1.show()